dss/events/handlers/index.py

```diff
@@ -16,48 +15,34 @@ def process_new_indexable_object(cls, event, logger) -> None:
 
     @classmethod
     def _process_new_indexable_object(cls, replica: Replica, key: str, logger):
-        if cls._is_bundle_to_index(key):
-            cls._index_and_notify(replica, key, logger)
-        elif cls._is_deletion(key):
-            cls._delete_from_index(replica, key, logger)
+        identifier = ObjectIdentifier.from_key(key)
+        if type(identifier) == BundleFQID:
```

Comment:
`isinstance(FQID)` would allow for polymorphism and is a more commonly used type check in Python. With non-distinct subtypes I would order the if branches accordingly from more specific to less specific. Just like the `except:` clause works in Python.

Reply:
✅